src/preload-hover.js

```diff
@@ -12,20 +12,23 @@ export default class PreloadHover {
   }
 
   start(domScope = this.configuration.defaultDomScope) {
-    if (!domScope) { throw new Error('domScope must be provided.'); }
+    if (!domScope) { throw new Error('domScope must be provided.'); } 
+    domScope = [...new Set(domScope)];
+    domScope.forEach(domScope => {
```

Comment:
Reusing the `domScope` variable in this instance should be avoided. The outer variable could be renamed `domScopes` to resolve this.